main.js

```diff
@@ -0,0 +1,102 @@
+const Game = function () {
+  let level = 0;
+  let time = 3;
+  let remaining = 0;
+  let timeVar;
+  let timeVare;
+  let isWin = true;
+
+  const randomColor = function () {
+    const r = Math.floor(Math.random() * Math.floor(255));
```

Comment:
try to name your variables something more descriptive. Even though `r` is short, short is not always better. You want someone to read through your code easily. Best code is clean and readable